From c8fcf794647e2a871b66ed9a4f7f815177794659 Mon Sep 17 00:00:00 2001
From: Dylan Reid <dgreid@chromium.org>
Date: Wed, 15 Feb 2017 15:41:22 -0800
Subject: [PATCH] Fix SSE4_2 compile path.

A semicolon has been missing from the "CHUNK" macro since it was
reformatted.

Change-Id: Ieb7d3b8166fda68bc8e15aec4a281c0af1019438
Signed-off-by: Dylan Reid <dgreid@chromium.org>
---
 src/support/city.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/support/city.c b/src/support/city.c
index d56498e22..997479093 100644
--- a/support/city.c
+++ b/support/city.c
@@ -494,7 +494,7 @@ static void CityHashCrc256Long(const char *s, size_t len, uint32 seed,
 		h = _mm_crc32_u64(h, c);				\
 		i = _mm_crc32_u64(i, d);				\
 		j = _mm_crc32_u64(j, e);				\
-		s += 40							\
+		s += 40;						\
 	} while (0)
 
 		CHUNK(1, 1);
-- 
2.11.1

